README.md

```diff
@@ -56,3 +56,11 @@ Finally, we want to run the migration script.
 - Open another terminal window.
 - Run `source .env`
 - Run `npx sequelize-cli db:migrate`
+
+## Lambda Functions and Step Function Workflows
+
+### Running locally
+
+1. Run `npm run dev:services` to spin up the local [AWS Step Functions emulator](https://docs.aws.amazon.com/step-functions/latest/dg/sfn-local-docker.html) provided by AWS.
```

Comment:
Just checking: Does this docker image need to be added to the `docker-compose.yml` file?

Reply:
yes, surprised isnt added here but it is added in the [future PRs](https://github.com/isomerpages/isomercms-backend/blob/feat/integrate-aws-with-site-launch-microservice/docker-compose.yml)!